### PR TITLE
Branch158 database documentation

### PR DESCRIPTION
**Pull request by TerrenceWiseman**
Need someone to review this and let me know what changes or clarification needs to be added.

### CODE REVIEW

File: documentation/database-documentation/database-schema.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#Interfaith Donation Database Schema
+
+##Tables
+* *Exammple for clarification will be donations needed for summer*
```
Comment (Tiernebre) [inline documentation/database-documentation/database-schema.md]: Typo, Exammple should be Example

File: documentation/database-documentation/database-schema.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#Interfaith Donation Database Schema
+
+##Tables
```
Comment (Tiernebre) [inline documentation/database-documentation/database-schema.md]: Add a space between the Markdown header and word (##Tables should be ## Tables)

File: documentation/database-documentation/database-schema.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#Interfaith Donation Database Schema
```
Comment (Tiernebre) [inline documentation/database-documentation/database-schema.md]: Add a space between the Markdown header and word (#Interfaith Donation Database Schema should be # Interfaith Donation Database Schema). This allows GitHub to properly show Markdown files.

**Review by Tiernebre — APPROVED** [review]

**Review by TylerNicholls — APPROVED** [review]
On the ER diagram I would change timestamp on donation to "donation_date" or something along those lines, but otherwise looks great!

**Comment by TerrenceWiseman** [comment]
made the changes :)

**Pull request merged**